flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/KubernetesJobManagerSpecification.java

```diff
@@ -30,10 +30,15 @@ public class KubernetesJobManagerSpecification {
 
     private List<HasMetadata> accompanyingResources;
 
+    private List<HasMetadata> prePreparedResources;
```

Comment:
```suggestion
    private final List<HasMetadata> prePreparedResources;
```

Reply:
hmm, how about just follow the style of `accompanyingResources`? If we change this, we should also keep the same with `accompanyingResources`